src/compile/data/bin.ts

```diff
@@ -15,7 +15,7 @@ export namespace bin {
     return `format('${format}', ${expr})`;
   }
 
-  function parse(model: Model): Dict<VgTransform[]> {
+  export function parseUnit(model: Model): Dict<VgTransform[]> {
```

Comment:
If you really need typing for `parseUnit` methods, I would prefer that you keep `parse` method the same but separately introduce function `parseUnit(model: UnitModel)` that call `parse(model: Model)` because semantically `parseFacet/Layer` DO NOT involve `parseUnit(facet/layerModel)` but rather a generic parse method.